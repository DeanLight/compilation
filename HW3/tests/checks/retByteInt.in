//retByteInt

int foo(byte me)
{
	return me;
}

byte foo2(int riguing)
{
	return riguing;
}

void main()
{
	return foo(4b);
}