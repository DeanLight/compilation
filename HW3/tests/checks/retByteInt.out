---end scope---
me BYTE -1
line 10: type mismatch
